builder/dockerfile/builder_test.go

```diff
@@ -0,0 +1,54 @@
+package dockerfile
+
+import (
+	"strings"
+
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/builder/dockerfile/parser"
+)
+
+func TestBuildProcessLabels(t *testing.T) {
+	dockerfile := "FROM scratch"
+	d := parser.Directive{}
+	parser.SetEscapeToken(parser.DefaultEscapeToken, &d)
+	n, err := parser.Parse(strings.NewReader(dockerfile), &d)
+	if err != nil {
+		t.Fatalf("Error when parsing Dockerfile: %s", err)
+	}
+
+	options := &types.ImageBuildOptions{
+		Labels: map[string]string{
+			"org.a": "cli-a",
```

Comment:
Yes, totally unnecessary but perhaps more clear to a gomature like myself. boy do I have a lot to learn! :smile: 

Many thanks for sharing, and for jumping on the label issue so quickly!  It's inspiring to witness this project work.